### ISSUE TITLE
[A11Y] Replace the qtip2 integration in the navigation toolbar

### ISSUE THREAD

**Issue by foxbunny** [indico/indico#5896]
**Describe the bug**

The navigation toolbar at the top of the page currently uses the qtip2 library to show the lower-level items. This results in code that is not quite accessible.

**To Reproduce**
1. Go to main page
2. Note that `a.arrow.js-dropdown` does not have appropriate ARIA attributes to indicate its state/purpose

**Expected behavior**

Expect `a.arrow.js-dropdown` to have `aria-controls` pointing to the menu items, and `aria-expanded` that reflects its state.

**Additional context**

- https://talk.getindico.io/t/ui-state-toolbar-navigation-menu/3216/7
- https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

**Comment by ThiefMaster** [comment]
@foxbunny I guess this one can be closed? (it was referenced in the header dropdown PR I merged, but not as `closes ...`)

**Comment by foxbunny** [comment]
Yes
